lib/src/values/small_logic_value.dart

```diff
@@ -41,12 +41,12 @@ class _SmallLogicValue extends LogicValue {
     return _masksOfWidth[width]!;
   }
 
-  const _SmallLogicValue(int value, int invalid, int width)
+  const _SmallLogicValue(int value, int invalid, super.width)
       : assert(width <= LogicValue._INT_BITS,
             '_SmallLogicValue should have low number of bits'),
         _value = ((1 << width) - 1) & value,
         _invalid = ((1 << width) - 1) & invalid,
-        super._(width);
+        super._();
```

Comment:
I will duplicate the explanation, because the commit comment is not so easy to find.

Here there is an implicit transfer of [super parameters](https://github.com/dart-lang/language/blob/master/accepted/2.17/super-parameters/feature-specification.md#examples).